metrics/grafana/pd.json

```diff
@@ -134,8 +134,9 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "delta(pd_tso_events{tidb_cluster=\"$tidb_cluster\", type=\"save\",instance=\"$instance\"}[1m]) > bool 0",
+          "expr": "pd_tso_role{tidb_cluster=\"$tidb_cluster\", instance=\"$instance\", dc=\"global\"}",
```

Comment:
This should be incompatible with metrics without global txn enabled, will it cause data to be unavailable under normal tso?

Reply:
It's compatible without Global TSO enabled.

Reply:
Normal TSO and Global TSO are the same things now. Global TSO will behave differently according to the cluster state.

Reply:
you mean If not enabled Global TSO, dc also set to `global`?

Reply:
Yes.